.github/workflows/test-stellar.yaml

```
@@ -0,0 +1,164 @@
+        with:
+          node-version: '18' # Hardcoded to ensure consistency.
+          cache: 'npm'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install dependencies
+        shell: bash
+        run: npm ci
+
+      - name: Install stable toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: 1.81.0
+          targets: wasm32-unknown-unknown
+
+      # install binary to avoid lengthy compilation process
+      - name: Install Stellar CLI
+        shell: bash
+        run: |
+          # Install Stellar CLI compatible with the soroban-sdk version in Cargo.toml
+          cargo install --locked stellar-cli --version 22.2.0 --features opt
+
+      - name: Setup Stellar local network
+        run: |
+          docker pull stellar/quickstart
+          docker run -d --rm -it \
+              -p 8000:8000 \
+              --name stellar \
+              stellar/quickstart:testing \
+              --local \
+              --enable-soroban-rpc \
+              --protocol-version 22
+
+          RPC_URL="http://localhost:8000/soroban/rpc"
+          NETWORK_PASSPHRASE="Standalone Network ; February 2017"
+          echo "RPC_URL=${RPC_URL}" >> $GITHUB_ENV
+          echo "NETWORK_PASSPHRASE=${NETWORK_PASSPHRASE}" >> $GITHUB_ENV
+
+      - name: Add as configured network
+        shell: bash
+        run: |
+          stellar network add local \
+              --rpc-url "$RPC_URL" \
+              --network-passphrase "$NETWORK_PASSPHRASE"
+
+      - name: Prepare local.json
+        run: |
+          echo '{
+              "chains": {
+                  "stellar": {
+                      "name": "Stellar",
+                      "axelarId": "stellar",
+                      "networkType": "local",
+                      "chainType": "stellar",
+                      "tokenSymbol": "XLM",
+                      "tokenAddress": "NATIVE_TOKEN_ADDRESS",
+                      "rpc": "http://127.0.0.1:8000/soroban/rpc",
+                      "horizonRpc": "http://127.0.0.1:8000",
+                      "contracts": {}
+                  }
+              }
+          }' > ./axelar-chains-config/info/local.json
+
+      - name: Compile WASM contracts
+        run: |
+          cd axelar-amplifier-stellar
+          stellar contract build
+          ./optimize.sh
+          cd ..
+          WASM_DIR=axelar-amplifier-stellar/target/wasm32-unknown-unknown/release
+          echo "WASM_DIR=${WASM_DIR}" >> $GITHUB_ENV
+
+      - name: Setup Stellar wallet
+        run: |
+          stellar keys generate test \
+              --rpc-url "$RPC_URL" \
+              --network-passphrase "$NETWORK_PASSPHRASE" \
+              --fund
+
+          STELLAR_PRIVATE_KEY=$(stellar keys show test)
+          STELLAR_ADDRESS=$(stellar keys address test)
+          echo "STELLAR_PRIVATE_KEY=${STELLAR_PRIVATE_KEY}" >> $GITHUB_ENV
+          echo "STELLAR_ADDRESS=${STELLAR_ADDRESS}" >> $GITHUB_ENV
```

```suggestion
          echo "STELLAR_PRIVATE_KEY=$(stellar keys show test)" >> $GITHUB_ENV
          echo "STELLAR_ADDRESS=$(stellar keys address test)" >> $GITHUB_ENV
```
remove intermediate variables